plugin_store/api/__init__.py

```diff
@@ -76,6 +77,12 @@ async def plugins_list(
     plugins = await db.search(db.session, query, tags, hidden)
     return plugins
 
+@app.post("/increment")
```

Comment:
We should accept plugin ID in the URL for proper RESTful endpoints. Maybe even a specific release? Tracking releases separately doesn't cost us more, and gives us more data in the future.

Reply:
Good idea. I'll check how easy/fast it is to sum a bunch of numbers for when we sort by total downloads later on, but I suppose we could just keep track of both total downloads and per-version downloads.

Reply:
Total can be just a sum of each version, done in DB, so we don't need to track them separately.

Reply:
Alright. Also turns out [that the point in the code](https://github.com/SteamDeckHomebrew/decky-loader/blob/42d08163feeb676e129fcfd8c9877aa3245da667/backend/browser.py#L178) where I'd like to put the request doesn't have the plugin ID, but it does have the version hash so this shouldn't be too difficult.